test/server/cards/01_SOR/events/MakeAnOpening.spec.js

```diff
@@ -16,17 +16,29 @@ describe('Make an Opening', function () {
                 });
             });
 
-            it('should reduce hp to a unit and heal your base', function () {
+            it('should fully-reduce hp to a unit and heal your base', function () {
```

Comment:
Does the test with isb agent cover anything that the test with at-st does not? If not, let's remove the isb agent test

Reply:
isb agent test is here to see if -2/-2 can go behind 0

Reply:
Oh gotcha. That's not needed here, we have other tests for power modifying effects which confirm this. So we can remove this test.

Reply:
He cover enemy unit while atst cover ally unit 😁